optimum/onnxruntime/model.py

```diff
@@ -49,6 +49,11 @@ def __init__(
             label_names (`List[str]`, `optional`):
                 The list of keys in your dictionary of inputs that correspond to the labels.
         """
+
+        logger.warning(
+            "The class `optimum.onnxruntime.model.ORTModel` is deprecated and will be removed in the next release."
```

Comment:
It shouldn't be used anywhere so no strong opinion on this, but we can remove it in v2.0 just to be safe @fxmarty wdyt ?

Reply:
No strong opinion on this either, it sounds fine to me to merge it now as well.